src/core/AutomatableModel.cpp

```diff
@@ -766,6 +774,20 @@ float AutomatableModel::globalAutomationValueAt( const MidiTime& time )
 	}
 }
 
+void AutomatableModel::setUseControllerValue(bool b)
+{
+	if (b)
+	{
+		m_useControllerValue = true;
+		emit dataChanged();
+	}
+	else if (m_controllerConnection && m_useControllerValue)
```

Comment:
No problem `m_useControllerValue` staying true, that is the default state before connecting any controllers.
The conditional is necessary because if there is no connected controller, the emission of [this](https://github.com/LMMS/lmms/pull/5657/files#diff-e9bc3866766f77eab76084d28eb3b8861574375e132403804d72e9a7ebf79fdeR787) signal is avoided, with the efficiency that this provides.

Reply:
Got it, I thought having it as true could affect some of the code that depends on it, but I guess every conditional that checks for `m_useControllerValue` also checks for `m_controllerConnection`. Resolving this!